v2/src/components/appInfoForm/index.tsx

```diff
@@ -33,14 +33,61 @@ type State = {
     netlifyApiRouteUsed: boolean,
     showWebsiteBasePath: boolean,
     showAPIBasePath: boolean
+    /** Whether current element is the first visible AppInfoForm */
+    firstAppInfoForm?: boolean
     // TODO: Add more fields here
```

Comment:
When clicking resubmit the user should be taken to the first visible instance of the form, but in this example the user is scrolled to a section that does not have the app info form (even though technically it does, it isnt visible and should not be considered the first instance on screen)

https://user-images.githubusercontent.com/18233774/187606064-5910b0c3-ca41-415b-b8de-a324453e8d7c.mp4

Reply:
To reproduce:

1. Navigate to frontend quick setup
2. In step 2) Fill the form for the React tab
3. Scroll to step 5)
4. Click resubmit

Reply:
fixed